spec/Records.tex

```diff
@@ -522,7 +434,7 @@ \subsection{Record Assignment}
 a default assignment operator for each record type \chpl{R} having the signature
 \begin{example}
 \begin{chapel}
-proc =(ref lhs:R, rhs:R) : void ;
+proc =(ref lhs:R, rhs) : void ;
 \end{chapel}
 \end{example}
 \noindent
```

Comment:
Independently of this PR, we should ask @bradcray for the rationale on whatever rules are currently implemented in this case.